cmd/assetsvc/handler.go

```diff
@@ -336,3 +379,25 @@ func newChartVersionListResponse(c *models.Chart) apiListResponse {
 
 	return cvl
 }
+
+func decodeParam(param string, w http.ResponseWriter) string {
+	decodedParam, err := url.PathUnescape(param)
+	if err != nil {
+		log.WithError(err).Errorf("could not decode param %s", decodedParam)
```

Comment:
you should be using `param` not `decodedParam` here, as if an error was returned then `decodedParam` will be an empty string, I would expect.